exercises/hangman/HangmanTest.cs

```diff
@@ -1,135 +1,342 @@
-﻿using Xunit;
-
-public class HangmanTest
+﻿using System;
+using Xunit;
+using Microsoft.Reactive.Testing;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Reactive.Concurrency;
+
+public class HangmanTests: ReactiveTest
 {
     [Fact]
-    public void Initially_9_failures_are_allowed()
+    public void Initial_state_masks_the_word()
     {
-        var game = new HangmanGame("foo");
+        // Arrange
+        Hangman hangman = new Hangman("foo");
```

Comment:
```suggestion
        var hangman = new Hangman("foo");
```

The convention in the test files is to use `var` whenever possible.